src/aks-preview/HISTORY.rst

```diff
@@ -2,6 +2,7 @@
 
 Release History
 ===============
+* [BREAKING CHANGE]: `az aks trustedaccess rolebinding create`: Remove deprecated `-r` and `-s` options
```

Comment:
Please move this paragraph down a few lines and choose a new version number with increased major version.

Reply:
And also update the value of `VERSION` in `setup.py`.

Reply:
Note, there's another breaking change #8217

Reply:
Thanks! updated!